### PR TITLE
Add a pseudo-random number function

### PR DESCRIPTION
**Pull request by Jmallone**
The pseudorandom number function in `kernel/lib/rand.cpp` folder.
`rand()` and `srand()`

### CODE REVIEW

File: kernel/lib/rand.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    _seedrand = _seedrand * 214013L + 2531011L;
+    return (unsigned int)(_seedrand >> 16) & MAX_RAND;
+}
+
+void srand(unsigned int seed)
+{
+    _seedrand = seed;
+}
+
+
```
Comment (Kfeavel) [inline kernel/lib/rand.cpp]: There's a bit of extra whitespace here that should be removed. Always have one empty line at the end of a file, but no more empty lines than that. Do the `fixup` technique that I talked to you about when you've made the change. This should be good practice. 😄

**Review by Kfeavel — CHANGES_REQUESTED** [review]

**Review by Kfeavel — DISMISSED** [review]
Looks good to me. Got everything rebased, and from what I can tell (on my phone), the whitespace is fixed. Letting @micahswitzer finish the review.

File: kernel/lib/rand.cpp (hunk 1 above)
Comment (micahswitzer) [reply kernel/lib/rand.cpp]: Both files now have one extra line. I'm not sure which editor you're using, but VIM will add an extra line past what's shown so maybe that's what happened here.

**Review by Jmallone — COMMENTED** [review]
Done, all empty line is over ;D

File: kernel/lib/rand.cpp (hunk 1 above)
Comment (Jmallone) [reply kernel/lib/rand.cpp]: i'ts done :D

**Review by micahswitzer — APPROVED** [review]
Great work! Thanks for contributing!

**Pull request merged**